c-cannons/bangazon-initial_site
# added all product types view and ability to click on individual products

#### Descriptions

User can view all product types when clicking on the 'All Categories' link under the 'Product Categories' dropdown on the navbar.

#### Steps to Test

1. User should see a list of all categories by name and the number of products available for sale in that category next to the name.

2. User should see no more than 3 products for each category

3. When clicking on a specific product, the user should be taken to that product page.

#### Link to Feature Ticket

#11
